tests/www/job_seekers_views/test_list.py

```diff
@@ -163,6 +166,284 @@ def test_multiple(client, snapshot):
+            <i class="ri-draft-line" aria-hidden="true" data-bs-toggle="tooltip"
+            data-bs-title="Postuler pour ce candidat">
+            </i>
+        </a>
+        """,
+        html=True,
+    )
+
+
+def test_job_seeker_created_by_prescriber_without_org(client):
+    """
+    Check that a job seeker created by an "orienteur solo" is not shared among
+    all the "orienteurs solo"
+    """
+    prescriber = PrescriberFactory()
+    other_prescriber = PrescriberFactory()
+    organization = PrescriberOrganizationWith2MembershipFactory()
+
+    # Job seeker created by another prescriber
+    alain = JobSeekerFactory(
+        first_name="Alain",
+        last_name="Zorro",
+        public_id="11111111-1111-1111-1111-111111111111",
+        post_code="29200",
+        city="Brest",
+        created_by=other_prescriber,
+    )
+    # Job seeker created by this prescriber
+    bernard = JobSeekerFactory(
+        first_name="Bernard",
+        last_name="Ygrec",
+        public_id="22222222-2222-2222-2222-222222222222",
+        post_code="29200",
+        city="Brest",
+        created_by=prescriber,
+    )
+    # Job seeker created by this prescriber when he was in an organization.
+    # He is not member of it anymore, this job seeker shouldn't be displayed.
+    charlotte = JobSeekerFactory(
+        first_name="Charlotte",
+        last_name="Xerus",
+        public_id="33333333-3333-3333-3333-333333333333",
+        post_code="29200",
+        city="Brest",
+        created_by=prescriber,
+        jobseeker_profile__created_by_prescriber_organization=organization,
+    )
+
+    client.force_login(prescriber)
+    response = client.get(reverse("job_seekers_views:list"))
+    parse_response_to_soup(response, selector="tbody")
+    assertNotContains(response, alain.get_full_name())
+    assertNotContains(response, reverse("job_seekers_views:details", kwargs={"public_id": alain.public_id}))
+    assertContains(
+        response,
+        f"""
+            <a href="{reverse("job_seekers_views:details", kwargs={"public_id": bernard.public_id})}?back_url={
+            reverse("job_seekers_views:list")
+        }" class="btn-link">{bernard.get_full_name()}</a>
+        """,
+        html=True,
+    )
+    assertContains(
+        response,
+        f"""
+        <a class="btn btn-sm btn-link btn-ico-only"
+            aria-label="Postuler pour ce candidat"
+            data-matomo-event="true" data-matomo-category="candidature" data-matomo-action="clic"
+            data-matomo-option="postuler-pour-ce-candidat"
+            id="introJsBtnPostuler"
+            href="{reverse("search:employers_results")}?job_seeker={bernard.public_id}&city={bernard.city_slug}">
+            <i class="ri-draft-line" aria-hidden="true" data-bs-toggle="tooltip"
+            data-bs-title="Postuler pour ce candidat">
+            </i>
+        </a>
+        """,
+        html=True,
+    )
+    assertNotContains(response, charlotte.get_full_name())
```

Comment:
Ce not contains me conforte dans le faite d'utiliser `soup.fin_all()` qui aura l'avantage de lister de manière exhaustive les utilisateurs présentés dans la page.